Insights/Host/HostIPStats.yaml

```diff
@@ -0,0 +1,349 @@
+SchemaVersion: '1.0'
+Type: KQL
+Provider: Sentinel
+DataTypes:
+  - DataType: Heartbeat
+  - DataType: VMConnection
+  - DataType: VMComputer
+  - DataType: ServiceMapComputer_CL
+  - DataType: WireData
+  - DataType: ProtectionStatus
+  - DataType: DeviceNetworkInfo
+  - DataType: DeviceNetworkEvents
+  - DataType: AzureNetworkAnalytics_CL
+  - DataType: DnsEvents
+  - DataType: CommonSecurityLog
+  - DataType: Event
+  - DataType: SecurityEvent
+  - DataType: Syslog
+EntitiesFilter: 
+ Host_OsFamily:
+  - Windows
+  - Linux
```

Comment:
I think it is better to leave this field blank. If left blank the Insight will run for any Host. If populated - the insight will not run if the OsFamily of a Host entity  is null (was not assigned any value).

Reply:
okay, will do.